swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/generator/parameters/PartParameter.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.swagger.generator.parameters;
+
+import io.swagger.models.parameters.AbstractSerializableParameter;
+import io.swagger.models.properties.FileProperty;
+
+public class PartParameter extends AbstractSerializableParameter<PartParameter> {
+  public PartParameter() {
+    super.setIn("formdata");
```

Comment:
we create a new "in" type for swagger
standard definition is "io.swagger.models.parameters.FormParameter"

i think it's better to use standard "in" type like the code i send to you.